.github/workflows/secpoll.yml

```diff
@@ -0,0 +1,25 @@
+---
+name: 'Verify secpoll zone syntax'
+
+on:
+  push:
+  pull_request:
+
+jobs:
```

Comment:
it would be nice to run this only on changes to the secpoll?

Reply:
It's only a 50 second job so I did not bother restricting it, but we could.